internal/datasources/app_endpoints_schema.go

```diff
@@ -114,7 +114,7 @@ func AppEndpointsSchema() schema.Schema {
 								},
 							},
 						},
-						"oidc": schema.SetNestedAttribute{
+						"oidc": schema.ListNestedAttribute{
```

Comment:
why change from list to set ?

Reply:
set to list

https://www.reddit.com/r/Terraform/comments/108dg3n/whats_the_difference_between_toset_and_tolist/
there is no guarantee that the items are unique, and there are loads of issues filed on terraform on SetNestedAttribute, given that the error we had suggests the schema needs to be changed, I experimented with List and found that it works.